bigtable/docs/snippets_table.py

```diff
@@ -387,5 +387,179 @@ def test_bigtable_table_row():
     table.truncate(timeout=300)
 
 
+def test_bigtable_column_family_name():
+    import re
```

Comment:
include re in sample.

Reply:
re is used for testing the output and is not part of sample, hence not included.
Moved re after END tag to avoid confusion.